terraform/user.tf

```diff
@@ -0,0 +1,15 @@
+# Create the test user
+module "user" {
+  source = "github.com/cisagov/molecule-iam-user-tf-module"
+
+  providers = {
+    aws                                    = aws.users
+    aws.images-production-provisionaccount = aws.images_production_provisionaccount
+    aws.images-staging-provisionaccount    = aws.images_staging_provisionaccount
+    aws.images-production-ssm              = aws.images_production_ssm
+    aws.images-staging-ssm                 = aws.images_staging_ssm
+  }
+
+  entity         = "ansible-role-cyhy-feeds"
+  ssm_parameters = ["/github/personal_authorization_token"]
```

Comment:
What is the PAT needed for here?

Reply:
The original repo had a Terraform test user, so I kept it.  You may be correct that the test user is no longer necessary since [cisagov/cyhy-feeds](https://github.com/cisagov/cyhy-feeds) is public.

Reply:
If we can't find any reason to keep the test user, then let's whack it.

Reply:
I'd like you guys to at least do a cursory confirmation before I re-skeletonize this skeletonization.

Reply:
I took a look and I didn't see any reason to need a test user here, but hopefully @mcdonnnj can double-check.

Reply:
I have modified this PR to re-skeletonize against [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role).